selfdrive/car/honda/carstate.py

```diff
@@ -10,7 +10,7 @@
 TransmissionType = car.CarParams.TransmissionType
 
 
-def get_can_signals(CP, gearbox_msg):
+def get_can_signals(CP, gearbox_msg, main_on_sig_msg):
```

Comment:
just call it `main_on_signal`, msg is confusing because usually that refers to the address the signal is on

Reply:
This variable is the message name, and the naming intends to convey that it's the name of the message containing the MAIN_ON signal.

Just as the existing `gearbox_msg` is e.g. `GEARBOX` (a message name), `main_on_sig_msg` is e.g. `SCM_FEEDBACK` (also a message name).

If it were called `main_on_signal`, then I would expect it to be a signal name such as `MAIN_ON`.

Reply:
my bad, then let's just call it `main_on_msg`

Reply:
(The situation with `SCM_FEEDBACK` and `SCM_BUTTON` messages is a little unusual, because *both* are actively used within each model, with signals like `MAIN_ON` and `DRIVERS_DOOR_OPEN` coming from one or the other based on model.)

Reply:
> my bad, then let's just call it main_on_msg

We agree about the `_msg` suffix, since it's a CAN message name.

I think it's useful to know whether we're referring to a message by the message name proper vs. the name of a signal contained within the message.   This little extra hint may save some confusion by a future person maintaining this code, or may reduce the need for comments.

More `get_can_signals()` args will likely be added-- please consider it.

Reply:
sounds good to me